tests/components/axis/test_binary_sensor.py

```diff
@@ -28,30 +28,32 @@ async def test_platform_manually_configured(hass):
     """Test that nothing happens when platform is manually configured."""
     assert (
         await async_setup_component(
-            hass, binary_sensor.DOMAIN, {"binary_sensor": {"platform": axis.DOMAIN}}
+            hass,
+            BINARY_SENSOR_DOMAIN,
+            {BINARY_SENSOR_DOMAIN: {"platform": AXIS_DOMAIN}},
         )
         is True
     )
 
-    assert axis.DOMAIN not in hass.data
+    assert AXIS_DOMAIN not in hass.data
 
 
 async def test_no_binary_sensors(hass):
     """Test that no sensors in Axis results in no sensor entities."""
     await setup_axis_integration(hass)
 
-    assert not hass.states.async_entity_ids("binary_sensor")
+    assert not hass.states.async_entity_ids(BINARY_SENSOR_DOMAIN)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)